papaya-web/src/main/java/pl/edu/agh/papaya/mappers/UserRoleMapper.java

```diff
@@ -0,0 +1,35 @@
+package pl.edu.agh.papaya.mappers;
+
+import org.springframework.stereotype.Component;
+import pl.edu.agh.papaya.api.model.UserRoleDto;
+import pl.edu.agh.papaya.model.UserRole;
+
+@Component
+public class UserRoleMapper {
+
+    public UserRole mapFromApi(UserRoleDto role) {
+        switch (role) {
+            case ADMIN:
+                return UserRole.ADMIN;
+
+            case MEMBER:
+                return UserRole.MEMBER;
+
+            default:
+                throw new IncompleteMappingException();
+        }
+    }
+
+    public UserRoleDto mapToApi(UserRole role) {
+        switch (role) {
+            case ADMIN:
+                return UserRoleDto.ADMIN;
+
+            case MEMBER:
+                return UserRoleDto.MEMBER;
+
+            default:
+                throw new IncompleteMappingException();
```

Comment:
Właśnie tu eksponujemy inne role niż mamy tak realnie - to `INACTIVE` nigdy nie może wyjść na zewnątrz, a jest valid stanem wedle aktualnej wizji

Reply:
Czyli stan INACTIVE ma wyjść na zewnątrz czy nie?

Reply:
Jak dla mnie nie

Reply:
No więc jest dobrze? Przez mapper nie przechodzi

Reply:
Ok, nie zagłębiałam się aż tak jeszcze - po prostu `IncompleteMappingException` mnie zbiło z tropu

Reply:
`IncompleteMappingException` z założenia ma być rzucone gdy nie da się czegoś przemapować.